### PR TITLE
Support latest terraform versions with newer library.

### PR DESCRIPTION
**Pull request by lorddaren**
The current python-terraform library was no longer being developed. The dda-python-library now supports the latest terraform versions and is being actively maintained.

### CODE REVIEW

**Comment by arm4b** [comment]
@bishopbm1 Could we get a review from @StackStorm-Exchange/encoretechnologies as a Terraform pack maintainer?

**Review by bishopbm1 — APPROVED** [review]
LGTM

**Comment by bishopbm1** [comment]
@Xorso looks like there were some build failures. We wont be able to push a new tag until resolved. Please look at: https://github.com/StackStorm-Exchange/stackstorm-terraform/runs/8230702319?check_suite_focus=true

**Pull request merged**
